keyboards/takashicompany/worm_eaten/worm_eaten.h

```diff
@@ -0,0 +1,12 @@
+// Copyright 2023 takashicompany (@takashicompany)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "quantum.h"
+
+#ifdef INTEGRATED
```

Comment:
```suggestion
#ifdef KEYBOARD_takashicompany_worm_eaten_integrated
```

Reply:
Correction.
Use SPLIT_KEYBOARD instead of the original #ifdef.